app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -43,65 +44,51 @@ const Notes = () => {
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add new note"
-            size="small"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                size="small"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
+    <>
+      <SideMenu showMenu={showMenuBar} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenuBar(!showMenuBar)}
+          title="All Notes"
+          actionBlock={
+            <Button
+              icon="ri-add-line"
+              label="Add new note"
+              size="small"
+              onClick={() => setShowNewNotePane(true)}
+            />
+          }
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+          }}
+        />
+        {notes.length ? (
```

Comment:
I understand this might work but let's be more explicit here. Being explicit sometimes goes a long way. Here adding the `> 0` part can make the code easily comprehensible.